paddle/fluid/framework/op_info.h

```diff
@@ -70,6 +70,11 @@ struct OpInfo {
     return grad_op_maker_;
   }
 
+  // some op has no grad_op_maker, can't use GradOpMaker() ?
+  bool HasGradOpMaker() const { return grad_op_maker_ != NULL ? true : false; }
```

Comment:
Use `nullptr` instead. This would be something more C++ 11 style.

Reply:
Done, thanks.